tools/scripts/features/frcnn/utils.py

```diff
@@ -0,0 +1,577 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Please remove the unnecessary functions from this. We should only keep the bare minimum that is required to run the script. Also the config and model files can be saved in our FB cluster and downloaded used from there similar to how the extract_features_vmb file is doing.

Reply:
Removed unnecessary functions, will upload model and config to AWS later